modules/migration-mtc-release-notes-1-8-3.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-3_{context}"]
+= {mtc-full} 1.8.3 release notes
+
+[id=technical-changes-1-8-3_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.3 has the following technical changes:
+
+.Support added for {oadp-short} 1.3
+
+{mtc-short} 1.8.3 adds support to {oadp-first} as a dependency of {mtc-short} 1.8.z. link:https://issues.redhat.com/browse/MIG-1520[(MIG-1520)]
+
+[id="resolved-issues-1-8-3_{context}"]
+== Resolved issues
+
+This release has the following major resolved issues:
+
+
+.Flaw in Golang `protobuf` module causes `unmarshal` function to enter infinite loop
```

Comment:
```suggestion
.CVE-2024-24786: Flaw in Golang `protobuf` module causes `unmarshal` function to enter infinite loop
```

Should this contain the CVE similar to how https://github.com/openshift/openshift-docs/pull/73356/files#diff-80c32a3c6e6095803a6894e98c9f19a1c1ade2b9017a26f84837b149f092005eR29 is written below?